kjourney/src/app/filter/filter.component.html

```diff
@@ -0,0 +1 @@
+<p>filter works!</p>
```

Comment:
do we need this

Reply:
not sure why it's appearing as a new file as it shouldn't be a part of my branch but should resolve itself when the ticket for the filter is merged